site/templates/nav.jet

```diff
@@ -2,73 +2,67 @@
 {{import "./nav/user_logged_in.jet"}}
 
 {{block nav(showSubNav=true)}}
-<div class="navigation" data-toggle="affix">
-  <nav class="navbar navbar-expand-md navbar-dark">
-      <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
-      
+<div class="navigation search-hidden" data-toggle="affix" aria-hidden="true">
+  <nav>
+
+    <div class="navbar">
+      <div class="navbar-brand-container">
+        <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
+      </div>
+
       {{if config("search_disabled") != "true"}}
-        <form class="form-search navbar-nav-search" action="{{ routeToPath("/search.html") }}">
-            <input class="form-control form-control-search" type="search" placeholder="{{i18n("search_control_placeholder")}}" name="q" aria-label="{{i18n("search_control_placeholder")}}">
-            <button class="sr-only" type="submit">{{i18n("search_control_submit")}}</button>
+        <form class="form-search navbar-nav-search search-desktop"  action="{{ routeToPath("/search.html") }}">
+          <button type="button" class="search-open" onclick="focusSearch()">
+            <svg class="search-icon" width="24" height="24" viewBox="0 0 24 24"><path d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z" /></svg>
+          </button>
+          <input class="form-control form-control-search" type="search" placeholder="Search" onfocusout="focusOutSearch()" name="q" aria-label="{{i18n("search_control_placeholder")}}">
+          <button class="search-close">
+            <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="24" height="24" viewBox="0 0 24 24"><path d="M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z" /></svg>
+          </button>
+          <button class="sr-only" type="submit">{{i18n("search_control_submit")}}</button>
         </form>
       {{end}}
 
-      <button class="navbar-toggler" type="button" toggle-function="toggleMobileMenu" s72-collapse=".navbar-collapse" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="icon-navbar-toggler"></span>
+      <button class="navbar-toggler" type="button" onClick="toggleHidden()" toggle-function="toggleMobileMenu" s72-collapse=".navigation" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Same issue as the other comment about calling functions on elements.  Also... I'm pretty sure `toggle-function="toggleMobileMenu"` has been broken since we added rollup.js to shift72-template so probably a good idea to look at alternative solutions.

Reply:
fixed